src/main/java/com/nike/vault/client/VaultClientFactory.java

```diff
@@ -114,13 +121,7 @@ public static VaultAdminClient getAdminClient() {
      * @return Vault admin client
      */
     public static VaultAdminClient getAdminClient(final UrlResolver vaultUrlResolver) {
-        return new VaultAdminClient(vaultUrlResolver,
-                new DefaultVaultCredentialsProviderChain(),
-                new OkHttpClient.Builder()
-                        .connectTimeout(DEFAULT_TIMEOUT, DEFAULT_TIMEOUT_UNIT)
-                        .writeTimeout(DEFAULT_TIMEOUT, DEFAULT_TIMEOUT_UNIT)
-                        .readTimeout(DEFAULT_TIMEOUT, DEFAULT_TIMEOUT_UNIT)
-                        .build());
+        return getAdminClient(vaultUrlResolver, new DefaultVaultCredentialsProviderChain());
```

Comment:
@tlisonbee It seems a little weird to have a chain of three methods called here. Why not just call call the three-parameter method with the default max requests here